Dockerfile

```diff
@@ -10,6 +10,8 @@ RUN apt update -y \
     curl python3 python3-pip sshpass \
     && rm -rf /var/lib/apt/lists/*
 
+RUN apt install -y vim nano vi
```

Comment:
Please add `vim` installation into line 10 to avoid more image layers instead of a new `RUN`.